### PR TITLE
Change package name

### PR DESCRIPTION
**Pull request by Evi1Pumpkin**

### CODE REVIEW

File: internal/features/modules/decoder/validations/unreferenced_origin.go [hunk 1]
```diff
@@ -43,14 +43,14 @@ func UnreferencedOrigins(ctx context.Context, pathCtx *decoder.PathContext) lang
 			// We temporarily ignore references with more than 2 segments
 			// as these indicate references to complex types
 			// which we do not fully support yet.
-			// TODO: revisit as part of https://github.com/hashicorp/terraform-ls/issues/653
+			// TODO: revisit as part of https://github.com/opentofu/opentofu-ls/issues/653
```
Comment (Gogotchuri) [inline internal/features/modules/decoder/validations/unreferenced_origin.go:46]: Let's leave terraform-ls issues links as they are.

File: internal/features/modules/decoder/validations/unreferenced_origin.go [hunk 2]
```diff
@@ -43,14 +43,14 @@ func UnreferencedOrigins(ctx context.Context, pathCtx *decoder.PathContext) lang
...
 			continue
 		}
 
 		// we only initially validate variables & local values
 		// resources and data sources can have unknown schema
 		// and will be researched at a later point
-		// TODO: revisit as part of https://github.com/hashicorp/terraform-ls/issues/1364
+		// TODO: revisit as part of https://github.com/opentofu/opentofu-ls/issues/1364
```
Comment (Gogotchuri) [inline internal/features/modules/decoder/validations/unreferenced_origin.go:53]: Same here

File: internal/features/modules/events.go [hunk 3]
```diff
@@ -258,7 +258,7 @@ func (f *ModulesFeature) decodeModule(ctx context.Context, dir document.DirHandl
 	// Changes to a setting currently requires a LS restart, so the LS
 	// setting context cannot change during the execution of a job. That's
 	// why we can extract it here and use it in Defer.
-	// See https://github.com/hashicorp/terraform-ls/issues/1008
+	// See https://github.com/opentofu/opentofu-ls/issues/1008
```
Comment (Gogotchuri) [inline internal/features/modules/events.go:261]: Issue link.

File: internal/features/modules/jobs/schema_test.go [hunk 4]
```diff
@@ -609,7 +609,7 @@ func TestPreloadEmbeddedSchema_basic(t *testing.T) {
 	cfgFS := fstest.MapFS{
 		// These are somewhat awkward double entries
 		// to account for io/fs and our own path separator differences
-		// See https://github.com/hashicorp/terraform-ls/issues/1025
+		// See https://github.com/opentofu/opentofu-ls/issues/1025
```
Comment (Gogotchuri) [inline internal/features/modules/jobs/schema_test.go:612]: Issue link

File: internal/features/modules/jobs/schema_test.go [hunk 5]
```diff
@@ -684,7 +684,7 @@ func TestPreloadEmbeddedSchema_unknownProviderOnly(t *testing.T) {
...
```
Comment (Gogotchuri) [inline internal/features/modules/jobs/schema_test.go:687]: Issue link

File: internal/features/modules/jobs/schema_test.go [hunk 6]
```diff
@@ -748,7 +748,7 @@ func TestPreloadEmbeddedSchema_idempotency(t *testing.T) {
 	cfgFS := fstest.MapFS{
 		// These are somewhat awkward two entries
 		// to account for io/fs and our own path separator differences
-		// See https://github.com/hashicorp/terraform-ls/issues/1025
+		// See https://github.com/opentofu/opentofu-ls/issues/1025
```
Comment (Gogotchuri) [inline internal/features/modules/jobs/schema_test.go:751]: Issue link

File: internal/features/modules/jobs/schema_test.go [hunk 7]
```diff
@@ -832,7 +832,7 @@ func TestPreloadEmbeddedSchema_raceCondition(t *testing.T) {
...
```
Comment (Gogotchuri) [inline internal/features/modules/jobs/schema_test.go:835]: Issue link

File: internal/features/rootmodules/jobs/lockfile_test.go [hunk 8]
```diff
@@ -119,7 +119,7 @@ func TestParseProviderVersions_multipleVersions(t *testing.T) {
 		},
 		// These are somewhat awkward two entries
 		// to account for io/fs and our own path separator differences
-		// See https://github.com/hashicorp/terraform-ls/issues/1025
+		// See https://github.com/opentofu/opentofu-ls/issues/1025
```
Comment (Gogotchuri) [inline internal/features/rootmodules/jobs/lockfile_test.go:122]: Issue link

**Review by Gogotchuri — CHANGES_REQUESTED** [review]
It's small, but we should leave the terraform-ls issue links unchanged.
I've added separate comments in a few places, for the same reason, but we should just replace /opentofu/opentofu-ls/issues with /hashicorp/terraform-ls/issues.
Other than that, everything is straightforward.

**Comment by Evi1Pumpkin** [comment]
Thanks @Gogotchuri, fixed it.

**Review by Gogotchuri — APPROVED** [review]

**Pull request merged**